_episodes/04-use-license.md

```diff
@@ -5,7 +5,185 @@ exercises: 0
+
+Since these are only three examples of issues that can arise, we recommend you to talk with the Intellectual Property (IP) office at your institution.
+
+Once added, a licence and all future versions of the software, will be under that licence. Most projects never need to change licenses, but occasionally circumstances change. When changing a project’s license, there is one fundamental thing to consider: **all** project's existing copyright holders should give their agreement in writing and signed. This is complicated because, without a previous copyright transfer agreement, the list of copyright holders can have thousands of names. We recommand to have a look at the [guide from GitHub on this question](https://opensource.guide/legal/#what-if-i-want-to-change-the-license-of-my-project) for more details.
+
+> ### Note: Dual License
+>
+> Some software that have comercial potential adopt a dual license model. They have a version of the code under the GNU General Public License that is freely available to everyone, but they also offer a proprietary version of the code in exchange of some cash. If you decide to adopt this business model, you must be very careful to correctly collect copyright transfer from all contributors to you.
+{: .callout}
+
+In principle, you can create your own bespoke licence, it's your right. However, we strongly discourage it, unless you have appropriate legal background. You are helping the users to understand the licence over your code or data by adopting an existing, widely described licence which better suits with your creation. So, users do not have to analyze the legal details of the licence to understand it.
+
+* Example licences explained in terms of the concepts discussed previously
+https://choosealicense.com/
+
+## Add a LICENCE file to a repository
+
+By including an open source license in your repository, it makes easier for other people to contribute. Most of the projects places their license information in a file named `LICENSE` (`LICENSE.txt` or `LICENSE.md` or `LICENSE.rst` or `COPYING`, `COPYING.txt`, `COPYING.md`, `COPYING.rst` are also used) in the root of the repository.
+
+> ## Challenge 
+> 
+> Add the previously selected license to your GitHub repository
+> 
+> > ## Solution
+> > 
+> > Via the GitHub interface for an already existing repository
+> > 1. On GitHub, navigate to the main page of the repository.
+> > 2. Click on `Create new file` button above the file list
+> > 3. Type `LICENSE` or `LICENSE.md` (with all caps) in the file name field
+> > 4. Click on `Choose a license template` on the right
+> > 5. Click on the selected license
+> > 6. Check that the license terms correspond to the one
+> > 7. Click on `Review and submit` button
+> > 8. Commit the file
+> > 
+> > Locally: 
+> > 1. Navigate to the root of the repository
+> > 2. Open your favorite text editor with a new file
+> > 3. Go to https://choosealicense.com
+> > 4. Select your license by opening it page
```

Comment:
opening it's page

Reply:
no apostrophe needed here - "opening its page" would be correct